pkg/publicapi/endpoint/shared/endpoint.go

```diff
@@ -76,7 +76,7 @@ func (e *Endpoint) id(c echo.Context) error {
 // @Description	each node is connected to a number of peer nodes.
 // @Tags			Utils
 // @Produce		json
-// @Success		200	{object}	[]peer.AddrInfo
+// @Success		200	{object}	[]string
```

Comment:
[peer.AddrInfo](https://pkg.go.dev/github.com/libp2p/go-libp2p@v0.32.2/core/peer#AddrInfo) is not a string.

Reply:
This is a mechanism to flatten a complex structure into a string - swaggo does this automatically. If we'd like to do this more thoroughly, we can, but I get it was overkill